test/known_problems/should_pass/inner_union_subtype_of_root_union.erl

```diff
@@ -0,0 +1,18 @@
+-module(inner_union_subtype_of_root_union).
+
+-export([f/0]).
+
+% The problem is that {t, a} | {t, b} is not subtype of {t, a|b}.
```

Comment:
We could add somewhere here that there's a similar case for map values: https://github.com/josefs/Gradualizer/blob/master/test/known_problems/should_pass/map_union.erl

Reply:
Aha! I'll merge them into a single file.

Reply:
Done.